torch/quantization/_quantize_script.py

```diff
@@ -133,5 +133,5 @@ def quantize_script(model, qconfig_dict, run_fn, run_args, inplace=False, debug=
         model = model.copy()
     return _quantize_script(model, qconfig_dict, run_fn, run_args, is_dynamic=False, debug=debug)
 
-def quantize_dynamic_script(model, qconfig_dict, run_fn, run_args, debug=False):
-    return _quantize_script(model, qconfig_dict, run_fn, run_args, is_dynamic=True, debug=debug)
+def quantize_dynamic_script(model, qconfig_dict, sample_model_inputs, debug=False):
+    return _quantize_script(model, qconfig_dict, None, sample_model_inputs, is_dynamic=True, debug=debug)
```

Comment:
can we provide some default arguments for `run_fn`, `run_args`, `is_dynamic`, `debug` in `_quantize_script` and pass `sample_model_inputs, is_dynamic=True, debug=debug` by keyword so we don't need to pass in `None` here?

Reply:
sure